getting-started/bare-metal/about.md

```diff
@@ -1,56 +1,53 @@
 canonical_url: '/getting-started/bare-metal/about'
 ---
 
 ### Big picture
 
-Secure hosts not in a cluster by installing {{site.prodname}} with networking and/or networking policy enabled.
+Secure non-cluster hosts by installing {{site.prodname}} for networking and/or networking policy.
 
 ### Value
 
-Not all hosts in your environment run virtualized workloads (i.e. containers managed by Kubernetes or OpenShift, or VMs managed by OpenStack). There may be physical machines or legacy applications that you cannot move into an orchestrated cluster that need to communicate securely with workloads in your cluster. Whether you have a thousand machines or ten, {{site.prodname}} lets you enforce policy on them using the same robust {{site.prodname}} network policy that is used for workloads. 
-
+Not all hosts in your environment run pods/workloads. You may have physical machines or legacy applications that you cannot move into a Kubernetes cluster, but still need to securely communicate with pods in your cluster. {{site.prodname}} lets you enforce policy on these **non-cluster hosts** using the same robust {{site.prodname}} network policy that you use for pods.
 
 ### Concepts
 
-#### Workloads and Hosts
-
-We use the term workload to mean a pod or VM running as a guest on a computer with {{site.prodname}} installed. If your cluster is a Kubernetes or OpenShift cluster, workloads are pods, if your cluster is an OpenStack cluster, workloads are VMs. 
+#### Non-cluster hosts and host endpoints
 
-We use the term host to mean a computer where {{site.prodname}} is installed. These include computers that are part of a cluster and “host” workloads, as well as computers that are not part of the cluster and run applications directly, which we call "non-cluster" hosts.  {{site.prodname}} does not handle the networking from host to host ({{site.prodname}} assumes this is set up), but it can be used to handle networking between hosts and workloads.  {{site.prodname}} can also provide network policy for hosts, regardless of whether or not the hosts run any workloads.  This guide focuses on non-cluster hosts.
+A **non-cluster host** is a computer that is running an application that is *not part of a Kubernetes cluster*. Using {{site.prodname}} network policy, you can secure these host interfaces, called **host endpoints**. Host endpoints can have labels, and work the same as labels on pods/workload endpoints. The advantage is, you can write network policy rules to apply to both workload and host endpoints using label selectors; where each selector can refer to the either type (or be a mix of the two). For example, you can write a cluster-wide policy for non-cluster hosts that is immediately applied to every host. To learn how to restrict traffic to/from hosts using {{site.prodname}} network policy see, [Protect hosts]({{site.baseurl}}/security/protect-hosts). 
```

Comment:
"called host endpoints" -> "using host endpoints"

The distinction is that a host endpoint can either be an in-cluster host interface, or an out-of-cluster host interface.

Reply:
> write a cluster-wide policy for non-cluster hosts

This seems to contradict itself. Maybe instead of "cluster-wide" you should say "global"?